controllers/gcpchaos/nodestop/types.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodestop
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/option"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/chaos-mesh/chaos-mesh/pkg/router"
+	ctx "github.com/chaos-mesh/chaos-mesh/pkg/router/context"
+	end "github.com/chaos-mesh/chaos-mesh/pkg/router/endpoint"
+)
+
+const (
+	GcpFinalizer = "gcp-Finalizer"
+)
+
+type endpoint struct {
+	ctx.Context
+}
+
+// GetComputeService is used to get the GCP compute Service.
+func GetComputeService(ctx context.Context, cli client.Client, gcpchaos *v1alpha1.GcpChaos) (*compute.Service, error) {
```

Comment:
how about moving this function to `controllers/gcpchaos` ?

Reply:
@WangXiangUSTC

Because the go file ("controllers/gcpchaos/*.go") has not been created so far, I just put it in the current location.

Reply:
But we can create one for it, it looks strange that `diskloss` import pkg `nodestop`.

Reply:
Hmm, let me revise it.